src/legacy/core_plugins/kibana/public/discover/top_nav/open_search_panel.js

```diff
@@ -21,14 +21,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { SavedObjectFinder } from 'ui/saved_objects/components/saved_object_finder';
 import rison from 'rison-node';
+import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 
 import {
   EuiSpacer,
   EuiFlyout,
```

Comment:
This should probably be in a modal not a since it's opening a saved object not adding one. But I can understand if that's out of scope. At the very least, can you apply the same suggestions from the dashboard add panel that I made to this one (adding FlyoutHeader and FlyoutFooter).